blocks/sharepoint/sharepoint.ts

```diff
@@ -282,20 +282,23 @@ export class Sharepoint {
         listId: string,
         folderId: string
       ) {
-        return await cbk.apiClient.msgraph
+        const msgraphClient = await cbk.apiClient.msgraph();
```

Comment:
@ianneo97 this can move it to the beginning of runtime function so that we don't need to redeclare same thing in each function